modules/gdscript/gdscript_editor.cpp

```diff
@@ -738,14 +738,17 @@ static void _get_directory_contents(EditorFileSystemDirectory *p_dir, HashMap<St
 
 static void _find_annotation_arguments(const GDScriptParser::AnnotationNode *p_annotation, int p_argument, const String p_quote_style, HashMap<String, ScriptLanguage::CodeCompletionOption> &r_result) {
 	if (p_annotation->name == SNAME("@export_range")) {
-		if (p_argument == 3 || p_argument == 4) {
+		if (p_argument == 3 || p_argument == 4 || p_argument == 5) {
 			// Slider hint.
 			ScriptLanguage::CodeCompletionOption slider1("or_greater", ScriptLanguage::CODE_COMPLETION_KIND_PLAIN_TEXT);
 			slider1.insert_text = slider1.display.quote(p_quote_style);
 			r_result.insert(slider1.display, slider1);
 			ScriptLanguage::CodeCompletionOption slider2("or_lesser", ScriptLanguage::CODE_COMPLETION_KIND_PLAIN_TEXT);
 			slider2.insert_text = slider2.display.quote(p_quote_style);
 			r_result.insert(slider2.display, slider2);
+			ScriptLanguage::CodeCompletionOption slider3("noslider", ScriptLanguage::CODE_COMPLETION_KIND_PLAIN_TEXT);
```

Comment:
Shouldn't the hint be `no_slider` for consistency with other things?

PS: This should have been discussed in a proposal, as it's not clear what are the use cases for hiding the slider.

Reply:
It was already exposed before and documented on the wiki, it was just not autocompletable. As for usecases there are situations where you only want a min value so you set the max value really high. In that case the slider is useless.

Yes no_slider would be more consistent, I will send a pr for it